website/docs/Pipelines/Genomic_Data_Commons_Whole_Genome_Somatic/README.md

```diff
@@ -38,79 +38,82 @@ For the latest workflow version and release notes, please see the [changelog](ht
-| gatk_applybqsr | [ApplyBQSR](https://gatk.broadinstitute.org/hc/en-us/articles/360037055712) | [GATK](https://gatk.broadinstitute.org/hc/en-us) | Recalibrates the base qualities of the input reads based on the recalibration table produced by the BaseRecalibrator tool and outputs a recalibrated BAM file; uses the emit_original_quals parameter to write the original base qualities under the BAM OQ tag |
-| collect_insert_size_metrics | [CollectInsertSizeMetrics](https://gatk.broadinstitute.org/hc/en-us/articles/360037055772) | [Picard](https://broadinstitute.github.io/picard/) | Generates metrics about insert size distribution in the form of both a histogram and table (txt file) |
+| sort_and_index_markdup_bam | sort | [Samtools](http://www.htslib.org) | Sorts and indexes the BAM file; outputs the sorted BAM and index file. |
+| [check_contamination](https://github.com/broadinstitute/warp/blob/master/tasks/broad/CheckContaminationSomatic.wdl) | [GetPileUpSummaries](https://gatk.broadinstitute.org/hc/en-us/articles/360037593451) and [CalculateContamination](https://gatk.broadinstitute.org/hc/en-us/articles/360036888972) | [GATK](https://gatk.broadinstitute.org/hc/en-us) | Summarizes counts of reads that support reference, alternate and other alleles for given sites and then calculates the fraction of reads coming from cross-sample contamination. |
+| gatk_baserecalibrator | [BaseRecalibrator](https://gatk.broadinstitute.org/hc/en-us/articles/360036898312) | [GATK](https://gatk.broadinstitute.org/hc/en-us) | Generates recalibration table for Base Quality Score Recalibration (BQSR). |
+| gatk_applybqsr | [ApplyBQSR](https://gatk.broadinstitute.org/hc/en-us/articles/360037055712) | [GATK](https://gatk.broadinstitute.org/hc/en-us) | Recalibrates the base qualities of the input reads based on the recalibration table produced by the BaseRecalibrator tool and outputs a recalibrated BAM file; uses the emit_original_quals parameter to write the original base qualities under the BAM OQ tag. |
+| collect_insert_size_metrics | [CollectInsertSizeMetrics](https://gatk.broadinstitute.org/hc/en-us/articles/360037055772) | [Picard](https://broadinstitute.github.io/picard/) | Generates metrics about insert size distribution in the form of both a histogram and table (txt file). |
 
-## Workflow Outputs
+## Workflow outputs
+The following table describes the workflow outputs. If running the workflow using Cromwell, these outputs will automatically be placed in the respective task execution directory.
 
-| Output Name | Description | Type |
+Alternatively, Cromwell allows you to specify an output directory using an options.json, as described in Cromwell's [Workflow Options Overview](https://cromwell.readthedocs.io/en/stable/wf_options/Overview/)(see the section on [Output Copying](https://cromwell.readthedocs.io/en/stable/wf_options/Overview/#output-copying)).
```

Comment:
So this information applies to every pipeline documented on the WARP doc site. It seems like a lot to copy the content in every single doc.. and a bit odd to only put it in this one (unless it's already in all the others?). Do we have a quickstart for running pipelines in Cromwell outside of Terra? If we do, it seems like this would be better to put there?

Reply:
Yes, I made a ticket to add it to each pipeline as I didn't think the lift would be that big. We don't currently have a quick start, but that sounds like a good idea too.